description: fix  unknown type name 'isc_boolean_t'
author: Christian Klein <cj-k@gmx.de>

Freetz: fixes the following build error

In file included from test.c:39:0:
../includes/omapip/omapip.h:268:1: error: unknown type name 'isc_boolean_t'
 isc_boolean_t omapi_protocol_authenticated (omapi_object_t *)

--- includes/omapip/omapip.h
+++ includes/omapip/omapip.h
@@ -34,6 +34,8 @@
 
 #include <dns/tsec.h>
 
+#include <isc/boolean.h>
+
 typedef unsigned int omapi_handle_t;
 
 struct __omapi_object;
